manuscript/recipes/mail.md

```diff
@@ -130,6 +130,8 @@ services:
       - internal
     deploy:
       replicas: 1
+      labels:
+        - traefik.enable=false
```

Comment:
This is only need if auto discovery is on.

Reply:
Which it is by default, and was in the old configuration.